### PR TITLE
Put validation queries and shapes into one folder; address omissions to release documentation

### LINKED ISSUE #711: Put validation files into one folder

### PR DESCRIPTION
**Pull request by rjyounes**

### CODE REVIEW

**Comment by rjyounes** [comment]
@Jamie-SA @marksem Could one of you have a quick look at this? The goal is to move the validation queries and shapes into one file. I've updated the paths in `bundle.yaml`, but what I'm unsure of is the changes on lines 78-82. There was no folder `{input}validations/` previously, only `{input}validation_queries`. I believe the target is where the validation reports go, but I'm not sure. I'm also wondering why, if this is correct, there is no target for the SHACL validations.

**Comment by rjyounes** [comment]
@Jamie-SA @marksem This probably slipped off your radar. I'd appreciate a review from one of you so we can merge it.

**Comment by Jamie-SA** [comment]
@rjyounes THis PR has conflicts to be resolved.
I will review.

File: bundle.yaml [hunk 1]
```diff
@@ -75,9 +75,9 @@ actions:
   includes:
     - gistCore.ttl
   stopOnFail: false
-  target: '{input}/validation'
+  target: '{input}/validation/queries/reports'
```
Comment (Jamie-SA) [inline bundle.yaml:78]: I wonder if putting the reports down into nested directories is a good idea. Wouldn't you want to make them obvious if there are errors?

**Review by Jamie-SA — APPROVED** [review]
There seems to be lots of changes showing that are unrelated to the PR. I'm guessing they are a result of merging in other work.

I don't see anything wrong with the bundle.yaml changes but the complexity of the onto_tool yaml makes it hard to review.

I ran onto_tool and it did produce a couple of warnings, I don't know if any of them are a problem:
- WARNING:root:gistDeprecated.ttl not found in .
- WARNING:root:README.md not found in ./docs
- WARNING:root:TimeRelatedChanges_v11.md not found in ./docs

File: bundle.yaml (hunk 1 above)
Comment (rjyounes) [reply bundle.yaml:76]: Good point. I moved them up a level into the validation directory.

**Comment by rjyounes** [comment]
@Jamie-SA 

I've expanded this PR by addressing your comment about missing folders. 

The PR now includes:

- Reorganizing validation queries and shapes.
- Consolidating `README.md` and `gistDocumentation.md` and deleting the latter, now redundant.
- Include the `README` in the release package `Documetation` directory in both `.md` and `.html` formats.
- Updates to `bundle.yaml` to accomplish these changes.

**Comment by rjyounes** [comment]
@Jamie-SA I've requested your re-review after recent changes.

**Comment by rjyounes** [comment]
@Jamie-SA OK, ready for your review now

**Review by dylan-sa — APPROVED** [review]
Updated README is looking good to me. Any issues w/ the yaml file will come out in the bundling process, and @rjyounes has already done some testing on this front.

File: bundle.yaml [hunk 2]
```diff
@@ -183,9 +183,21 @@ actions:
...
-    - "gistDocumentation.md"
-    - "README.md"
-    - "TimeRelatedChanges_v11.md"
+- action: "copy"
+  message: "Copying readme to Documentation directory and patching local links."
+  source: "README.md"
+  target: "{output}/Documentation/README.md"
+  replace:
+    from: "\\(./(\\w+)\\.md\\)"
+    to: "(./\\g<1>.html)"
```
Comment (Jamie-SA) [inline bundle.yaml:192]: I'm not real sure what this is doing, but it doesn't seem necessary when the next action is translating the `.md` file to `.html`.

Comment (rjyounes) [reply bundle.yaml:192]: Yes, I've removed that since you looked at it.

**Pull request merged**